test/dlblob_test.js

```diff
@@ -11,10 +11,10 @@
 const assert = require('assert');
 const sinon = require('sinon');
 const nock = require('nock');
-const alcollector = require('al-collector-js');
+const alcollector = require('@alertlogic/al-collector-js');
```

Comment:
If we're going to reference al-collector-js directly in the code. I would add it as a direct dependency in package.json.

Reply:
Agree. But it is referenced in tests only. Therefore I'd say we need to rework tests in order no to reference `al-collector-js`, but this is a separate big change.